### ISSUE TITLE
Portals Publisher and Admin cannot view in Microsoft Edge 44.18362.449.0

### ISSUE THREAD

**Issue by erangatl** [wso2/product-apim#9010]
### Description:
Publisher and Admin portals are not working in Microsoft Edge 44.18362.449.0, Devportal is working. 
All working when remote viewing a server hosted on mac.

![Publisher](https://user-images.githubusercontent.com/20179540/89794751-1df45680-db45-11ea-801a-5afec72066c5.png)

![Admin](https://user-images.githubusercontent.com/20179540/89794773-26e52800-db45-11ea-8a68-614b56a8a9e1.png)

![deveportal](https://user-images.githubusercontent.com/20179540/89794782-29e01880-db45-11ea-8f07-f253de1bb03f.png)

### Affected Product Version:

RC3

### Environment details (with versions):
- OS: Windows 10
- Client:
- Env (Docker/K8s):
- Microsoft Edge 44.18362.449.0

---

**Comment by tmkasun** [comment]
Fixed in https://github.com/wso2/carbon-apimgt/pull/9159

The cause of this issue was, the TextEncoder function is not available in edge < 79 versions, which is used in [hapi joi](https://github.com/sideway/joi/issues/2229#issuecomment-558074834) validation library. We had to add a polyfill as mentioned in [here](https://github.com/sideway/joi/issues/2229#issuecomment-558074834).
